homeassistant/components/motionmount/entity.py

```diff
@@ -42,12 +42,28 @@ def __init__(self, mm: motionmount.MotionMount, config_entry: ConfigEntry) -> No
                 (dr.CONNECTION_NETWORK_MAC, mac)
             }
 
+    @property
+    def available(self) -> bool:
+        """Return True if the MotionMount is available (we're connected)."""
+        return self.mm.is_connected
+
+    def update_name(self) -> None:
+        """Update the name of the associated device."""
+        if TYPE_CHECKING:
```

Comment:
You need to add `from typing import TYPE_CHECKING` on file level

Reply:
Sorry, normally Git complains if I forget such a thing, but this time I committed your suggestion using GitHub.
I fixed it now